### DISCUSSION TITLE
Welcome to health_rules_engine Discussions!

### DISCUSSION THREAD

**Discussion by ToferC** [ToferC/health_rules_engine#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomcdona** [comment]
I like the idea of standardizing a schema at a higher abstraction than an sql table/view. It's crazy however the number of different versions of a "Person" I've come across in the schemas, or a "traveller" or a "contact". Common taxonomy to start would be super beneficial. Does a "public health lexicon" currently exist? 

Or maybe there's something on [schema.org](https://schema.org/docs/schemas.html) already we could adopt?

**Reply by ToferC** [reply]
This is super @tomcdona. Thanks! I hadn't actually seen schema.org! I went through and updated the major models based on the schema and Rust syntax. Also a bunch more updates done and hope to dig in a bit deeper this weekend.
